src/control/cmd/daos/container.go

```diff
@@ -351,6 +353,86 @@ func (cmd *containerCreateCmd) Execute(_ []string) (err error) {
 	return nil
 }
 
+type containerSerializeCmd struct {
+	daosCmd
+
+	Output string `long:"output-path" short:"o" description:"path to output serialized HDF5 files" required:"0"`
+	Args   struct {
+		Source string `positional-arg-name:"<Source container>"`
+	} `positional-args:"yes"`
+}
+
+func (cmd *containerSerializeCmd) Execute(_ []string) error {
+	ap, deallocCmdArgs, err := allocCmdArgs(cmd.log)
+	if err != nil {
+		return err
+	}
+	if cmd.Args.Source == "" {
```

Comment:
`Source` here refers to daos://pool/cont or the UNS path. I think it might make more sense to say "Source is required", which would prompt the user to look at the `--help`. Although that doesn't display the daos://pool/cont format. In fact, I see `daos fs copy` and `daos cont clone` don't note this format either, although at one point they did